scripts/ccf/member/add.sh

```diff
@@ -5,43 +5,13 @@
 
 REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../..")"
 
-ccf-member-add() {
-    set -e
-
-    source $REPO_ROOT/scripts/ccf/az-cleanroom-aci/setup.sh && az-cleanroom-aci-setup
-    source $REPO_ROOT/scripts/ccf/propose.sh
-    source $REPO_ROOT/scripts/ccf/member/info.sh
-    source $REPO_ROOT/scripts/ccf/member/id.sh
-    source $REPO_ROOT/scripts/ccf/member/use.sh
+ccf-member-add-gov() {
```

Comment:
Can you add a brief comment on what it does? `-gov` part confused me at first, so a comment would help readers to understand it quickly

Reply:
Sure will do

Reply:
Added in https://github.com/microsoft/azure-privacy-sandbox-kms/pull/435/commits/53644fddc2b27f8523ecdf812a70ed452a6b68b3